JustSaying.IntegrationTests/FluentNotificationStackTestBase.cs

```diff
@@ -11,6 +11,7 @@
 
 namespace JustSaying.IntegrationTests
 {
+#pragma warning disable CA1054
```

Comment:
It would be better to put the pragmas immediately above the specific lines they apply to, and then turn them on again immediately after. Otherwise you're turning off the entire rule for all the code in the entire file.

```csharp
#pragma warning disable BadThing
// Code that isn't really bad
#pragma warning restore BadThing
```

Otherwise someone knows you're turning a rule off, but doesn't know specifically what specific code is causing the warning.

Reply:
Is the "restore" needed if it comes at the bottom of the file?

Reply:
If the error is on a class being declared, then you'd do this:

```csharp
#pragma warning disable BadThing
public class Foo
#pragma warning restore BadThing
```

You wouldn't wrap the entire class, because it isn't every line that causes the warning, just the first.